pkg/acceptance/cluster/docker.go

```diff
@@ -262,7 +291,7 @@ func (c *Container) Wait(ctx context.Context, condition container.WaitCondition)
 	case err := <-errCh:
 		return err
 	case waitOKBody := <-waitOKBodyCh:
-		outputLog := filepath.Join(c.cluster.logDir, "console-output.log")
+		outputLog := filepath.Join(c.cluster.volumesDir, "logs", "console-output.log")
```

Comment:
Does this mean we'll henceforth also put the data directories into the artifacts? Maybe that's ok, but explain it in the commit message.